test/MsgPackTest.php

```diff
@@ -30,6 +31,24 @@ public function test_00_msgpack_call() {
             ]
         ]);
         $this->assertEquals($resp[0][0], 2);
+
+        $check_call_17 = self::$tarantool->call('tarantool_version_at_least',
+                                                array(1,7,2,0));
+        if ($check_call_17[0][0]) {
+            self::$tarantool->set_modern_call_mode(true);
+            $resp = self::$tarantool->call('test_4', [
+                '4TL2tLIXqMqyGQm_kiE7mRrS96I5E8nqU', 'B627', 0, [
+                    'browser_stats_first_session_hits' => 1
+                ]
+            ]);
+            $this->assertEquals($resp[0], 2);
+            $resp = self::$tarantool->call('test_4', [
+                '4TL2tLIXqMqyGQm_kiE7mRrS96I5E8nqU', 'B627', 0, [
+                    'browser_stats_first_session_hit' => 1
+                ]
+            ]);
+            $this->assertEquals($resp[0], 2);
+        }
```

Comment:
`@dataProvider` annotation allows to reduce duplication in such cases. There is example in `CreateTest.php`.

Reply:
After some discussion, we decided that KISS will our choice in this case.